Added test case to lock in behaviour (#48541)
pandas-dev/pandas: doc/source/whatsnew/v1.6.0.rst

ExtensionArray
^^^^^^^^^^^^^^
- Bug in :meth:`Series.mean` overflowing unnecessarily with nullable integers (:issue:`48378`)

> - Loss of dtype information when concatenating EA with empty EA of same type (:issue:`48510`)

Reviewer: `EA` -> `ExtensionArray`

Also could you make "Loss of dtype" more specific to describe that the result dtype is object?

> - When concatenating DataFrame containing an empty ExtensionArray Series with another DataFrame with an ExtensionArray of the same type, the resulting dtype turned into object (:issue:`48510`)

Reviewer: ```suggestion
- Bug when concatenating an empty DataFrame with an ExtensionDtype to another DataFrame with the same ExtensionDtype, the resulting dtype turned into object (:issue:`48510`)
```
